share/plugins/colornotes.qml

```diff
@@ -74,18 +74,18 @@ MuseScore {
                         if (fullScore)
                               cursor.rewind(0) // if no selection, beginning of score
 
-                        while (cursor.segment && (fullScore || cursor.tick < endTick)) {
-                              if (cursor.element && cursor.element.type === Element.CHORD) {
-                                    var graceChords = cursor.element.graceNotes;
+                        while (cursor.segment() && (fullScore || cursor.tick < endTick)) {
+                              if (cursor.element() && cursor.element().type == Ms.CHORD) {
```

Comment:
my recent change used a `===`

Reply:
That has changed on purpose... at least, the fact that the element types now have a different namespace, ie, Ms.CHORD and not Element.CHORD. (Not my idea, I'm building off previous work)
I don't see that the == vs === makes an important difference here.

Reply:
QtCreator hinted me at that it does make a (subtle) difference, that's why I changed it.